src/foam/core/Property.js

```diff
@@ -593,9 +593,16 @@ foam.CLASS({
       return prop;
     },
 
-    function exportAs(obj) {
+    function exportAs(obj, path) {
```

Comment:
for exporting 'abc.a.b.c', we walk path to create this.abc$.dot(a).dot(b).dot(c)